storage-proofs-porep/src/stacked/circuit/params.rs

```diff
@@ -137,12 +136,12 @@ impl<Tree: MerkleTreeTrait, G: 'static + Hasher> Proof<Tree, G> {
         // -- verify replica column openings
 
         // Private Inputs for the DRG parent nodes.
-        let mut drg_parents = Vec::with_capacity(layers);
+        let num_layers = drg_parents_proofs[0].len();
+        let mut drg_parents = Vec::with_capacity(num_layers);
 
         for (i, parent) in drg_parents_proofs.into_iter().enumerate() {
             let (parent_col, inclusion_path) =
                 parent.alloc(cs.namespace(|| format!("drg_parent_{}_num", i)))?;
-            assert_eq!(layers, parent_col.len());
```

Comment:
These asserts aren't totally necessary, but I still think they're useful as a sanity check that the Prover's witness (column proofs) was constructed correctly. Circuits can be hard to debug after they've been synthesized (e.g. an error during Groth16 proving because the Groth16 parameters don't match the synthesized circuit) and I've found that these `assert`s during synthesis can save debugging time later on.

Reply:
I've removed it as `layers` and `parent_col.len()` is always true for the first item, as `layers` is `drg_parents_proofs[0].len()`. But I taken into account, that of course the other proofs could potentially be wrong. So I'm adding it back.

Reply:
Cool, thanks!